### PR TITLE
Mention playwright robottest browser setup for local testing

### PR DESCRIPTION
**Pull request by petschki**
----
📚 Documentation preview 📚: https://plone6--1772.org.readthedocs.build/

### CODE REVIEW

File: docs/contributing/core/index.md [hunk 1]
```diff
@@ -248,9 +248,17 @@ It takes 5-10 minutes to run the full unit test suite.
 If you run acceptance tests with the `--all` option, it will run tests in a real browser.
 This takes 30-40 minutes to run.
 This may repeatedly launch and close browser windows that gain focus, disrupting you from doing any other work.
-If this happens, you can install the `chromedriver` OS package.
-See https://developer.chrome.com/docs/chromedriver.
-Then run `export ROBOT_BROWSER="headlesschrome"` and again run `bin/test --all`.
+If this happens, you can use `headlesschrome` as testbrowser by setting an environment variable `export ROBOT_BROWSER="headlesschrome"` and again run `bin/test --all`.
```
Comment (stevepiercy) [inline docs/contributing/core/index.md]:
````suggestion
If this happens, you can use `headlesschrome` as the test browser.
First set an environment variable.

```shell
export ROBOT_BROWSER="headlesschrome"
```

Then run all tests again.

```shell
bin/test --all
```
````

File: docs/contributing/core/index.md [hunk 2]
```diff
@@ -248,9 +248,17 @@ It takes 5-10 minutes to run the full unit test suite.
...
+
+Since we use [Playwright](https://playwright.dev/) for our robottests you have to install it first.
```
Comment (stevepiercy) [inline docs/contributing/core/index.md]:
```suggestion
Plone uses [Playwright](https://playwright.dev/) to run robot tests.
```

File: docs/contributing/core/index.md [hunk 3]
```diff
@@ -248,9 +248,17 @@ It takes 5-10 minutes to run the full unit test suite.
...
+`plone.app.robotframework` provides a script for that:
```
Comment (stevepiercy) [inline docs/contributing/core/index.md]:
```suggestion
`plone.app.robotframework` provides a script to install Playwrite browsers.
```

File: docs/contributing/core/index.md [hunk 4]
```diff
@@ -248,9 +248,17 @@ It takes 5-10 minutes to run the full unit test suite.
...
+
+```shell
+# Install playwright browsers
```
Comment (stevepiercy) [inline docs/contributing/core/index.md]:
```suggestion
```

File: docs/contributing/core/index.md [hunk 5]
```diff
@@ -248,9 +248,17 @@ It takes 5-10 minutes to run the full unit test suite.
...
+./bin/rfbrowser init
+```
+
+After the browser resources are downloaded and initalized you can run the tests.
 
 ```shell
 # Run acceptance tests
```
Comment (stevepiercy) [inline docs/contributing/core/index.md]:
```suggestion
```

File: docs/contributing/core/index.md [hunk 6]
```diff
@@ -248,9 +248,17 @@ It takes 5-10 minutes to run the full unit test suite.
...
+
+Since we use [Playwright](https://playwright.dev/) for our robottests you have to install it first.
+`plone.app.robotframework` provides a script for that:
+
+```shell
+# Install playwright browsers
+./bin/rfbrowser init
+```
+
+After the browser resources are downloaded and initalized you can run the tests.
```
Comment (stevepiercy) [inline docs/contributing/core/index.md]:
```suggestion
After the script downloads and initalizes browser resources, you can run the acceptance tests.
```

**Review by stevepiercy — CHANGES_REQUESTED** [review]
I'll make the changes. Thanks for the PR! This is great!

**Review by stevepiercy — APPROVED** [review]

**Comment by stevepiercy** [comment]
Refs: https://github.com/plone/Products.CMFPlone/pull/4048#issuecomment-2472319057

**Comment by stevepiercy** [comment]
Linkcheck is a temporary failure. Merging. Thank you!

File: docs/contributing/core/index.md (hunk 3 above)
Comment (petschki) [reply docs/contributing/core/index.md]: small typo: it should be `Playwright`

Comment (stevepiercy) [reply docs/contributing/core/index.md]: Thanks, will fix.

Comment (stevepiercy) [reply docs/contributing/core/index.md]: Fixed in https://github.com/plone/documentation/pull/1776/commits/3ac55229e9264b198eda284d8341028df8e89f01 in https://github.com/plone/documentation/pull/1776

**Pull request merged**